test/machineslib.py

```diff
@@ -290,3 +290,9 @@ def setUp(self):
 
         # HACK: older c-ws versions always log an assertion, fixed in PR cockpit#16765
         self.allow_journal_messages("json_object_get_string_member: assertion 'node != NULL' failed")
+
+    def tearDown(self):
+        b = self.browser
+        if (b.is_present("#button.alert-link.more-button")):
```

Comment:
This also needs to check `b.cdp.valid`, so that it doesn't try to do that if a test fails in the setup phase, before starting the browser and logging into cockpit.

Reply:
Fixed in #627